_api/api_look4

```diff
@@ -0,0 +1,93 @@
+---
+title: API Look4 Géoportail
+tagline: la recherche géographique multithématique du Géoportail - adresses, lieux, cadastre, PR etc
+contract: OUVERT sous contrat
+owner: IGN Géoportail
+category: Public 
+layout: api
+logo: img/look4-geoportail.jpg
+contact: contact.geoservices@ign.fr
+doc_tech: https://ignf.github.io/look4/latest/jsdoc/
+access_link: http://professionnels.ign.fr/ign/contrats
+clients:
+  - développeurs
+  - entreprises
+  - organismes publics
+  - collectivités territoriales
+  - ministères
+keywords:
+  - cadastre
+  - adresse
+  - lieux-dits habités
+  - lieux-dits inhabités
+  - points routiers
+---
+
+# API Look4 Géoportail  
+##  recherche géographique multithématique - adresses, lieux, cadastre, PR etc
+
+Look4 est une API de recherche “nouvelle génération” proposée par l’IGN qui fournit une interface unique à l’utilisateur pour de multiples sources de données. Look4 sert les besoins du site Géoportail, mais est disponible pour tout site ou application.
+
+Look4 permet, en un seul outil, de faire des recherches sur de nombreuses sources de données géolocalisées préconfigurées, telles que des adresses, des parcelles, des lieux-dits, des contenus du Géoportail, du Géocatalogue ou de data.gouv.fr, ainsi que des index spécialisés tels que les points de repère routiers. 
+
+Look4 permet donc une recherche puissant et multidimensionnelle avec une interface simple.
+
+Les sources de données proposées par Look4 peuvent être de deux natures :
+
+- des sources indexées directement par Look4 ;
+- des web services de recherche tiers existants.
+Ces sources s’enrichissent progressivement : il est possible de solliciter l’IGN (en écrivant à l’adresse contact.geoservices@ign.fr) pour en rajouter de nouvelles.
+
+##Comment s’utilise Look4 ?
+Look4 propose à l’utilisateur une API REST en JSON qui fournit des méthodes pour :
+
+- connaître les données mises à disposition de l’utilisateur (API de découverte) ;
+- effectuer des recherches dans les données selon les critères de l’utilisateur (API de recherche).
+
+L’accès à Look4 se fait par les URL suivantes : https://wxs.ign.fr/VOTRE_CLE_API/look4/user/discover (API de découverte) et https://wxs.ign.fr/VOTRE_CLE_API/look4/user/search (API de recherche)
+
+La documentation complète de l’API est disponible à la page : https://ignf.github.io/look4/latest/jsdoc
+
+
+Look4 est disponible pour test sur la clé “beta” de l’IGN : https://wxs.ign.fr/beta/look4/user/discover et https://wxs.ign.fr/beta/look4/user/search.
```

Comment:
Les URL en beta/look4/usr/search ne fonctionnent pas

Reply:
faute de frappe, devrait être "user".

Reply:
Erreur dans mon commentaire... my bad

Sur l'exemple de la doc: https://wxs.ign.fr/beta/look4/user/search?indices=localisant&method=fuzzy&match[fulltext]=73%20av%20de%20paris%20saint-mand%C3%A9

J'ai un "No rights for this ressource or ressource does not exist"

Reply:
Pas retrouvé cet exemple, mais je crois que le contexte montrait que ce n'était qu'indicatif : les ressources sont données par https://wxs.ign.fr/look4/user/discover/ (oui, il se trouve qu'il y a une ressource qui s'appelle "locating", donc dès qu'on retrouve on clarifiera un peu)